packages/hardhat/hardhat.config.ts

```diff
@@ -1,7 +1,14 @@
+import * as dotenv from "dotenv";
+dotenv.config();
 import { HardhatUserConfig } from "hardhat/config";
 import "@nomicfoundation/hardhat-toolbox";
 import "hardhat-deploy";
 
+// ToDo: add default api key
+const defaultApiKey = process.env.ALCHEMY_API_KEY ?? '';
+// ToDo: auto generate mnenoic for the deployer account
+const deployerPrivateKey = process.env.DEPLOYER_PRIVATE_KEY ?? '';
```

Comment:
'' => ""

We definitely need to have linting (with prettier) on the hardhat folder also. We could create a issue for it.

Reply:
created new issue https://github.com/scaffold-eth/se-2/issues/26